style.css

```diff
@@ -0,0 +1,493 @@
+    color: var(--third-color-medium);
+}
+h6 {
+    font-size: 16px;
+}
+.style-one {
+    color: var(--third-color-medium);
+    font-weight: 900;
+}
+.style-two {
+    color: var(--second-color-dark);
+}
+.style-three {
+    color: var(--second-color-dark);
+    font-size: 20px;
+    font-weight: 540;
+}
+.style-four {
+    margin: 0px 0px 0px 10px;
+    padding: none;
+}
+.line-height {
+    line-height: 1.5rem;
+}
+/* DASHED BOXES 
+.dashed-border-black {
+    border: 2px black dashed;
+    margin: 2px;
+}
+.dashed-border-blue {
+    border: 2px blue dashed;
+    margin: 2px;
+}
+.dashed-border-green {
+    border: 2px green dashed;
+    margin: 2px;
+}
+.dashed-border-pink {
+    border: 2px pink dashed;
+    margin: 2px;
+}
+*/
+/* GRIDS */
+/* parents */
+.parent-container {
+    display: grid;
+    grid-template-columns: 1fr 4fr 1.8fr;
+}
+.main-container {
+    display: grid;
+    grid-template-rows: var(--header-height) 170px 20px auto;
+
+    width: auto;
+    overflow-x: auto;
+    max-height: 100vh;
+    overflow-y: scroll;
+    scrollbar-width: none; 
+    -ms-overflow-style: none;
+}
+.main-container::-webkit-scrollbar { 
+    width: 0;
+    height: 0;
+}
+.whats-happening-box {
+    display: grid;
+    grid-template: 
+    "profile-pic whats-happening-form whats-happening-form" auto
+    ". icons-list-home tweet-button-light-small" 50px /
+    80px 1fr 110px
+    ;
+}
+.left-menu-container {
+    display: grid;
+    grid-template-rows: var(--header-height) 570px 50px 1fr 55px;
+
+    height: 100vh;
+    max-height: 100vh;
+}
+.right-side-bar-container {
+    grid-template-rows: var(--header-height) minmax(400px, auto) 400px;
+    max-height: 100vh;
+    overflow-x: auto;
+    max-height: 100vh;
+    overflow-y: scroll;
+    scrollbar-width: none; 
+    -ms-overflow-style: none;
+}
+.right-side-bar-container::-webkit-scrollbar { 
+    width: 0;
+    height: 0;
+}
+.tweet-icon-holder {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    max-width: 800px;
+}
+.tweet-icon-holder img {
+vertical-align: middle;
+}
+.tweet {
+    display: grid;
+    grid-template: 
+    "retweet-icon retweet-name ." auto
+    "profile-pic-small tweet-content three-dots-icon" auto
+    ". tweet-icon-holder ." 50px /
+    50px 1fr 50px
+}
+/* columns */
+.two-columns__expand-two {
+    display: grid;
+    grid-template-columns: auto 1fr;
+}
+.two-columns__expand-one {
+    display: grid;
+    grid-template-columns: 1fr auto;
+}
+.three-columns__expand-two {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+}
+.five-columns {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+}
+/* rows */
+.two-rows {
+    display: grid;
+    grid-template-rows: 1fr 1fr;
+}
+.two-rows__expand-one {
+    display: grid;
+    grid-template-rows: 1fr auto;
+}
+.two-rows__expand-two {
+    display: grid;
+    grid-template-rows: auto 1fr;
+}
+.three-rows__expand-two {
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+}
```

Comment:
I like these .class names, they can be reused again when needed in other parts of the project.